acp/main_module.php

```diff
@@ -233,27 +237,7 @@ protected function purge_textformatter_cache()
 	 */
 	protected function purge_mediaembed_cache()
 	{
-		try
-		{
-			$iterator = new \DirectoryIterator($this->cache->cache_dir);
-		}
-		catch (\Exception $e)
-		{
-			return;
-		}
-		foreach ($iterator as $fileInfo)
-		{
-			if ($fileInfo->isDot() || $fileInfo->isDir())
-			{
-				continue;
-			}
-
-			$filename = $fileInfo->getFilename();
-			if (strpos($filename, 'http.') === 0)
-			{
-				$this->cache->remove_file($fileInfo->getPathname());
-			}
-		}
+		$this->purge_cache->mediaembed_purge_cache();
```

Comment:
I think it'd make more sense looking like:
```
$this->mediaembed_cache->purge();
```
So name the object mediaembed_cache and the method should just be purge.